src/modules/getLikes.js

```diff
@@ -0,0 +1,25 @@
+const LikeAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/0SbroHwilqPhRaF9QpdF/likes';
+
+const sendLikes = (name) => {
+  fetch(LikeAPI, {
+    method: 'POST',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      item_id: name,
+    }),
+  });
+};
```

Comment:
[OPTIONAL] As I mention in the following comment, this function should be asynchronous like the other one so they wait for each other before completing.